src/main/java/com/sba/hockeyGame/domain/utils/ApiUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.sba.hockeyGame.domain.utils;
+
+import com.sba.hockeyGame.domain.model.Player;
+import java.util.List;
+import javax.management.InstanceAlreadyExistsException;
+
+public class ApiUtils {
+
+  public static void checkTeamCaptainAlreadyExist(final Long yearOfTeam, final boolean isCaptain, final List<Player> players)
+      throws InstanceAlreadyExistsException {
+    if (isCaptain &&
+        players.stream()
+            .anyMatch(player -> Boolean.TRUE == player.getIsCaptain())) {
```

Comment:
the lambda expression cannot be simplified by method reference like 
```suggestion
            .anyMatch(player::getIsCaptain)) {
```

Reply:
good advice